plugins/bundle/plugin_test.go

```diff
@@ -429,25 +429,27 @@ func TestPluginOneShotDeltaBundle(t *testing.T) {
 	ids, err := manager.Store.ListPolicies(ctx, txn)
 	if err != nil {
 		t.Fatal(err)
-	} else if len(ids) != 1 {
-		t.Fatal("Expected 1 policy")
+	}
+	if len(ids) != 1 {
+		t.Fatalf("Expected 1 policy, got %d", len(ids))
 	}
 
 	bs, err := manager.Store.GetPolicy(ctx, txn, ids[0])
-	exp := []byte("package a\n\ncorge=1")
 	if err != nil {
 		t.Fatal(err)
-	} else if !bytes.Equal(bs, exp) {
+	}
+	exp := []byte("package a\n\ncorge=1")
+	if !bytes.Equal(bs, exp) {
 		t.Fatalf("Bad policy content. Exp:\n%v\n\nGot:\n\n%v", string(exp), string(bs))
 	}
 
 	data, err := manager.Store.Read(ctx, txn, storage.Path{})
-	expData := util.MustUnmarshalJSON([]byte(`{"a": {"baz": "bux", "foo": ["hello", "world"]}, "system": {"bundles": {"test-bundle": {"etag": "foo", "manifest": {"revision": "delta", "roots": ["a"]}}}}}`))
-
 	if err != nil {
 		t.Fatal(err)
-	} else if !reflect.DeepEqual(data, expData) {
-		t.Fatalf("Bad data content. Exp:\n%v\n\nGot:\n\n%v", expData, data)
+	}
+	expData := util.MustUnmarshalJSON([]byte(`{"a": {"baz": "bux", "foo": ["hello", "world"]}, "system": {"bundles": {"test-bundle": {"etag": "foo", "manifest": {"revision": "delta", "roots": ["a"]}}}}}`))
+	if !reflect.DeepEqual(data, expData) {
+		t.Fatalf("Bad data content. Exp:\n%#v\n\nGot:\n\n%#v", expData, data)
```

Comment:
This was a small, but good fix. :+1: 

The `err` check is for the `Store` a few lines up, and `MustUnmarshalJSON` doesn't return an error, so an independent check for its error condition (wrong result) is put in a few lines down. This is clearer than the "chained-if" style that was there before, I think.

Reply:
I think that's something a linter should be able to catch, anyways. If the if's branch breaks the control flow (like, `return`, or `t.Fatal` or `panic`), then there's no need to wrap the other conditionals into `else`. But maybe it's just a pet peeve of mine.